DataFormats/Detectors/TRD/include/DataFormatsTRD/Constants.h

```diff
@@ -51,10 +51,12 @@ constexpr int NCOLMCM = 18;     // the number of pads per MCM
 
 constexpr int NBITSTRKLPOS = 11;                   // number of bits for position in tracklet64 word
 constexpr int NBITSTRKLSLOPE = 8;                  // number of bits for slope in tracklet64 word
-constexpr float PADGRANULARITYTRKLPOS = 40.f;      // TODO this should come out of the trapconfig.
-constexpr float PADGRANULARITYTRKLSLOPE = 1000.f;
+constexpr int ADDBITSHIFTSLOPE = 1 << 3;           // in the TRAP the slope is shifted by 3 additional bits compared to the position
```

Comment:
I guess the original TODO comment still applies: this should come out of the trapconfig.

Reply:
Ok, I will put the TODO comment back in place. Implementing this will require also changes in the TrackletTransformer which needs to know the granularities for slope and position. Or I could also change this now, should I better do that?